code/obj/item/material.dm

```diff
@@ -882,19 +882,40 @@
 
 		playsound(src.loc, sound_process, 40, 1)
 
+	proc/load_reclaim(obj/item/W as obj, mob/user as mob)
+		. = FALSE
+		if (istype(W,/obj/item/raw_material/) || istype(W,/obj/item/sheet/) || istype(W,/obj/item/rods/) || istype(W,/obj/item/tile/) || istype(W,/obj/item/cable_coil) || istype(W,/obj/item/wizard_crystal))
+			W.set_loc(src)
+			if (user) user.u_equip(W)
+			W.dropped()
+			. = TRUE
+
 	attackby(obj/item/W as obj, mob/user as mob)
 		if (W.cant_drop) //For borg held items
 			boutput(user, "<span class='alert'>You can't put that in [src] when it's attached to you!</span>")
 			return ..()
-		if (istype(W,/obj/item/raw_material/) || istype(W,/obj/item/sheet/) || istype(W,/obj/item/rods/) || istype(W,/obj/item/tile/) || istype(W,/obj/item/cable_coil) || istype(W,/obj/item/wizard_crystal))
+		if (istype(W,/obj/item/storage/) || istype(W,/obj/item/satchel/))
+			var/obj/item/storage/S = W
+			var/obj/item/satchel/B = W
+			for(var/obj/item/O in W)
```

Comment:
A very edge case scenario is using a bible on the reclaimer. In that case the items are not stored in the bible itself and as such this won't work. You should consider iterating over `S.get_contents()` instead of just `S` if  `W` is of a storage type.
```suggestion
			var/items = W
			if(istype(S))
				items = S.get_contents()
			for(var/obj/item/O in items)
```